src/Common/Core/Impl/Install/RInstallation.cs

```diff
@@ -322,10 +407,10 @@ public static class RInstallation {
                 foreach (IFileSystemInfo fsi in directories) {
                     string subFolderName = fsi.FullName.Substring(baseRFolder.Length + 1);
                     Version v = GetRVersionFromFolderName(subFolderName);
-                    if (v.Major >= minMajorVersion &&
-                        v.Minor >= minMinorVersion &&
-                        v.Major <= maxMajorVersion &&
-                        v.Minor <= maxMinorVersion) {
+                    if (v.Major >= supportedVersions.MinMajorVersion &&
```

Comment:
This check is incorrect. Consider min version as 3.2, max version as 5.0. Then 4.1 will not match, because 1 is not greater than 2, and is not less than 0.

I think we should just use `Version` everywhere, then we can use its own comparison operators.